tests/test_edtfdatestring.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test the marshmallow EDTF Date String."""
+
+import pytest
+from edtf.parser.grammar import ParseException, level0Expression
+from marshmallow import Schema, ValidationError, fields
+
+from marshmallow_utils.fields import EDTFDateString
+
+
+class MySchema(Schema):
+    f = EDTFDateString()
+
+
+@pytest.mark.parametrize('test_input, expected',
+                         [("2020-01-02garbage", "2020-01-02"),
+                          ("2020-01-45", "2020-01")])
+def test_deserialize_assert(test_input, expected):
+
+    assert MySchema().load({'f': test_input}) == {'f': expected}
+
+
+@pytest.mark.parametrize('test_input_a',
+                         [("2020-08/1998-08"),
+                          ("12-040-400")])
+def test_deserialize_raiset(test_input_a):
+    pytest.raises(
+        ValidationError,
+        MySchema().load,
+        {'f': test_input_a}
+    )
```

Comment:
I think it fits in one line (check if its not longer than 79):
```suggestion
    pytest.raises(ValidationError, MySchema().load, {'f': test_input_a})
```